### ISSUE TITLE
Languages

### ISSUE THREAD

**Issue by tomwrobel** [tomwrobel/ora_data_model#6]
Languages should be mapped from the ISO 639-1 two digit code to the ISO language name

**Comment by AndrewBennet** [comment]
The default `language` field in Elements is actually just a Text field - it isn't guaranteed to be a ISO 639-1 code. If there _was_ a restricted-list field configured in Elements holding all the language codes, we could include a value-mapping in the xwalk to map between ISO codes and ISO names.

**Comment by tomwrobel** [comment]
OK, so we may have to do this at our end after ingest and exchange it as a text field?

A note for the interested: we screwed up one of our forms when we implemented select lists for languages and managed - invisibly to the user - to select the first language on the list if not language was selected (it was not a mandatory field).

We were later contacted by a researcher using our OAI-PMH feed asking if we really did have 10,000 records in Afar, a minority language in Djibouti, Eritrea and Somalia with fewer than two million native speakers. If so, it would have represented the largest digital corpus of that language. We had to disappoint them.
